packages/lesswrong/lib/collections/comments/schema.ts

```diff
@@ -131,16 +132,16 @@ const schema = {
   pageUrl: resolverOnlyField({
     type: String,
     canRead: ['guests'],
-    resolver: (comment, args, context) => {
-      return context.Comments.getPageUrl(comment, true)
+    resolver: (comment, args, context: ResolverContext) => {
+      return Comments.getPageUrl(comment, true)
```

Comment:
Not an issue for the PR, but I don't currently know why/when one should use "context.Comments" vs "Comments".

Reply:
Reasons to use `context.Comments` (in a resolver):
 * It has `loaders`, which are a cache that's not shared between requests
 * It doesn't involve an import, so it doesn't create import cycles
Reasons to use `Comments` (a top-level import):
 * They don't require a `ResolverContext`
 * The type system captures the fact that functions have been monkeypatched onto it